packages/storage-documents/src/item-type.ts

```diff
@@ -5,5 +5,6 @@ export enum ItemType {
     page = 'page',
     pageScanResult = 'pageScanResult',
     issueScanResult = 'issueScanResult',
-    pageScanRunResult = 'pageScanRunResult',
+    onDemandPageScanRunResult = 'pageScanRunResult',
+    OnDemandPageScanRequests = 'pageScanRequests',
```

Comment:
lower case - it seems lint rules are broken, it is possible to have unused vars and imports now